timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -384,6 +386,38 @@ def _get_hpo_train_fn_kwargs(self, **train_fn_kwargs) -> dict:
         """
         return train_fn_kwargs
 
+    def _is_gpu_available(self) -> bool:
+        return False
+
+    def hyperparameter_tune(
+        self, hyperparameter_tune_kwargs="auto", hpo_executor: HpoExecutor = None, time_limit: float = None, **kwargs
+    ):
+        if hpo_executor is None:
+            hpo_executor = self._get_default_hpo_executor()
+            default_num_trials = kwargs.pop("default_num_trials", None)
+        hpo_executor.initialize(
```

Comment:
In `core.models.abstract.AbstractModel.hyperparameter_tune` this line is inside the `if` condition, why do we need to change it here?

Reply:
Thanks for catching this!